packages/backend/src/config/parseConfig.mock.ts

```diff
@@ -45,8 +45,10 @@ export const LOCAL_PROJECT_MISSING_PROJECT_DIR = {
 };
 
 export const LOCAL_PROJECT_MISSING_PROFILES_DIR = {
-    ...LOCAL_PROJECT,
-    profiles_dir: undefined,
+    type: 'dbt',
+    name: 'project',
+    profiles_dir: 'hello',
```

Comment:
should this prop be removed ( based on the var name )

Reply:
or the variable should be named `LOCAL_PROJECT_MISSING_PROJECT_DIR`

Reply:
Yep missed something when merging these changes. Thanks!